classes/Form.php

```diff
@@ -841,6 +841,9 @@ public function post()
                 if ($field['type'] === 'checkbox' || $field['type'] === 'switch') {
                     $data[$name] = isset($data[$name]) ? true : false;
                 }
+                if ($field['type'] === 'checkboxes' && !isset($data[$name])) {
```

Comment:
It's a hack, but.. so is the checkbox if from the above.

Reply:
Well, but it looks correct to me. Beside that we need to fix the checkbox (singular) in [another issue](https://github.com/getgrav/grav-plugin-form/issues/476), the checkboxes (plural) code should be correct.